cob_base_velocity_smoother/package.xml

```diff
@@ -8,13 +8,16 @@
   <url type="website">http://ros.org/wiki/cob_base_velocity_smoother</url>
   <!-- <url type="bugtracker"></url> -->
 
-  <maintainer email="frm@ipa.fhg.de">Florian Mirus</maintainer>
+  <maintainer email="bnm@ipa.fhg.de">Benjamin Maidel</maintainer>
   <author email="frm@ipa.fhg.de">Florian Mirus</author>
```

Comment:
a general question, just noticed it here:
how do we handle people having left IPA in terms of email addresses?
I think it is possible to remove the emails from the `author` tags, but not from the `maintainer` tags.

So, I suggest to remove the email address here, as it is not valid anymore...